CHANGELOG.md

```diff
@@ -7,212 +7,190 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
```

Comment:
Since this file is auto-generated, it should be excluded from the formatter and linters.